libraries/nginx_service.rb

```diff
@@ -94,6 +96,7 @@ def action_enable
       directory '/run/nginx' do
```

Comment:
I'd be surprised if nginx needs to run as root. We likely just need to change the permissions on the service account's home directory (or whatever directory the process chroot).